packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -278,6 +278,10 @@ const AppStack = createStackNavigator(
     // Note, WalletHome isn't in this stack because it's part of the tab navigator
     [Screens.TabNavigator]: { screen: TabNavigator },
     [Stacks.SendStack]: { screen: SendStack },
+    [Screens.SendConfirmation]: { screen: SendConfirmation },
```

Comment:
I assume you've added this here so that clicking back from payment request goes back to home. If so, can you add a comment mentioning that?

Reply:
Yes, you are right. Will do!